modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-test/src/testFunctional/tests/ProcessBuilderKaleoDesignerReact.testcase

```diff
@@ -901,12 +901,26 @@ definition {
 	}
 
 	@description = "Verify that is possible to delete translations"
-	@ignore = "Test Stub"
 	@priority = "3"
 	test CanDeleteTranslations {
+		property test.name.skip.portal.instance = "ProcessBuilderKaleoDesignerReact#CanDeleteTranslations";
+
+		Workflow.openWorkflowListView();
+
+		Workflow.addNewDefinition(workflowDefinitionTitle = "New Workflow Definition");
 
-		// TODO CanDeleteTranslations pending implementation
+		ProcessBuilderKaleoDesignerReact.selectTranslationLanguage(translationOptionSelected = "en-us");
+
+		ProcessBuilderKaleoDesignerReact.viewTranslationLanguagePresentOnDropdownMenu(translationOption = "fr_FR");
+
+		ProcessBuilderKaleoDesignerReact.gotoManageTranslations(
+			deleteLanguage = "French",
+			done = "true",
+			searchLanguage = "French");
+
+		ProcessBuilderKaleoDesignerReact.selectTranslationLanguage(translationOptionSelected = "en-us");
 
+		ProcessBuilderKaleoDesignerReact.viewTranslationLanguageNotPresentOnDropdownMenu(translationOption = "fr_FR");
```

Comment:
You should no go to the translation language dropdown to view if the language it's no longer present, this is not the goal of this testcase, you could assert if the languages deletes are no longer present on the manage translations component.